src/Umbraco.Web/Redirects/RedirectUrlSearchResults.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Umbraco.Core.Models;
+
+namespace Umbraco.Web.Redirects
+{
+    public class RedirectUrlSearchResult
+    {
+        public string StatusMessage { get; set; }
+        public IEnumerable<IRedirectUrl> SearchResults { get; set; }
```

Comment:
This cannot return this model, `IRedirectUrl` is a business logic model, this is used for persistence not for view models.